scripts/rook.sh

```diff
@@ -1,14 +1,33 @@
 #!/bin/bash -e
 
-ROOK_VERSION=${ROOK_VERSION:-"v1.1.7"}
+ROOK_VERSION=${ROOK_VERSION:-"v1.2.7"}
 ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
 ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
+ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
```

Comment:
please add an empty line between variables and the next function

Reply:
done.